### PR TITLE
Update index.md to include AutoMLPipeline

### PR DESCRIPTION
**Pull request by ppalmes**

### CODE REVIEW

**Comment by bkamins** [comment]
Thank you for the contribution. Let us just wait for a language review before merging.

File: docs/src/index.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ integrated they are with DataFrames.jl.
     - [ScikitLearn.jl](https://cstjean.github.io/ScikitLearn.jl/stable/):
       A Julia wrapper around the full Python scikit-learn machine learning library.
       Not well integrated with DataFrames.jl, but can be combined using StatsModels.jl.
+    - [AutoMLPipeline](https://github.com/IBM/AutoMLPipeline.jl):
+      is a package that makes it trivial to create complex ML
```
Comment (nalimilan) [inline docs/src/index.md:71]:
```suggestion
      A package that makes it trivial to create complex ML 
```

**Review by nalimilan — COMMENTED** [review]
I have (somewhat surprisingly :-) very few suggestions to make.

**Review by bkamins — APPROVED** [review]

**Comment by ppalmes** [comment]
thanks all!

**Pull request merged**